macaroon/Classes/Views/ListComponents/CollectionCell.swift

```diff
@@ -41,3 +51,14 @@ open class CollectionCell<ContextView: ViewComposable>: UICollectionViewCell, Li
         contextView.prepareForReuse()
     }
 }
+
+extension CollectionCell {
+    private func addSeperatorView() {
+        switch seperatorStyle {
+        case .none:
+            return
+        case let .single(seperator):
```

Comment:
I think the name `single` is not sufficient to explain it is for adding a separator to the bottom of a cell. `.singleBottom(separator)` would be more explanatory here.

Reply:
Nop it should be named as `singleLine` since iOS has similar thing on its API, `UITableViewCell.SeperatorStyle.singleLine` .

Also we decided that only bottom seperators will be supported. So no need a distinction.